protos/dcs/tts/v0/tts.proto

```diff
@@ -34,11 +34,11 @@ message TransmitRequest {
 
   // The origin of the transmission. Relevant if the SRS server has "Line of
   // Sight" and/or "Distance Limit" enabled.
-  dcs.common.v0.Position position = 5;
+  Position position = 5;
```

Comment:
I've noticed that since I've added `u` and `v` to `dcs.common.v0.Position `, it is very inconvenient to use it as an input, which is why I've changed this to a Position that only has `lat`/`lon`/`alt`. Not pretty though 🤷‍♂️

Reply:
Hmmm, that is a good point. `Position` is used all over the place as it is a built-in DCS'ism... I wonder if we need to rename your enhanced `Position` to something else to differentiate.

Reply:
I think the inclusion of `u`/`v` makes sense for everything we return. So we might have to split it based on input vs output. We could have `dcs.common.v0.Position` for output, and something like `LatLonPosition`/`LatLonPosition`/`InputPosition`/`RequestPosition` as the type for requests?

Unless, we want to provide the choice that position can be provided either via `lat/lon/alt` or via `u/v/alt`. For the Lua API, the latter might be easier to use, as you usually work with `u/v/alt` in the Lua part (that is basically `x/z/y` from the position object you get from DCS).

Reply:
Option 1: What if we kept a single position but made everything apart from lat/lon/alt optional?
Option 2: I think `InputPosition` as you suggested above is probably the best compromise

Reply:
Option 1: I am not a fan due to:
- requires `null` checks as a client even though it is always included
- could be provided in requests, but wouldn't have an effect

I'll go with Option 2

Reply:
I've implemented Option 2 in [a285bad](https://github.com/DCS-gRPC/rust-server/pull/202/commits/a285bad38d37fdd5a63d757bf5f95149f3f2fd80). It doesn't feel pretty, but the only other option I had was to use a structure similar to (which isn't ideal either; similar to your `DetailedPosition` suggestion, just with reusing a base `Position`):

```proto
message Position {
  double lat = 1;
  double lon = 2;
  double alt = 3;
}

message ExtendedPosition {
  Position position = 1;
  double u = 4;
  double v = 5;
}
```

If you prefer that, I am happy to update the PR again.

Reply:
The position with u/v is returned so much (All the unit stream events for starters, right?) that I think having the extra `Position` inside of `ExtendedPosition` is a bit icky. I think the option 2 is the best compromise here.